src/js/tabs/pid_tuning.js

```diff
@@ -2192,3 +2298,177 @@ TABS.pid_tuning.updatePIDColors = function(clear = false) {
+                $('#pid-tuning .pid_titlebar .rate').text(i18n.getMessage("pidTuningRcRateRaceflight"));
+                $('#pid-tuning .pid_titlebar .rc_expo').text(i18n.getMessage("pidTuningRcExpoKISS"));
+
+                break;
+
+            case 3: // Actual
+                $('#pid-tuning .pid_titlebar .rc_rate').text(i18n.getMessage("pidTuningRcRateActual"));
+                $('#pid-tuning .pid_titlebar .rate').text(i18n.getMessage("pidTuningRateQuickRates"));
+                $('#pid-tuning .pid_titlebar .rc_expo').text(i18n.getMessage("pidTuningRcExpoRaceflight"));
+                
+                rate_max = 2.0;
+                
+                if (!sameRatesType) {
+                    rc_rate_default = (0.2).toFixed(2), rate_default = 0.67, expo_default = 0.54;
+                }
+
+                break;
+
+            case 4: // QuickRates
+                $('#pid-tuning .pid_titlebar .rc_rate').text(i18n.getMessage("pidTuningRcRate"));
+                $('#pid-tuning .pid_titlebar .rate').text(i18n.getMessage("pidTuningRateQuickRates"));
+                $('#pid-tuning .pid_titlebar .rc_expo').text(i18n.getMessage("pidTuningRcExpoRaceflight"));
+                
+                rate_max = 2000;
+                rate_step = 10;
+                
+                if (sameRatesType) {
+                    $('.pid_tuning input[name="pitch_rate"]').val((RC_tuning.pitch_rate * 1000).toFixed(0));
+                    $('.pid_tuning input[name="roll_rate"]').val((RC_tuning.roll_rate * 1000).toFixed(0));
+                    $('.pid_tuning input[name="yaw_rate"]').val((RC_tuning.yaw_rate * 1000).toFixed(0));
+                } else {
+                    rate_default = 670;
+                }
+
+                break;
+
+            // add future rates types here
+            default: // BetaFlight
+                $('#pid-tuning .pid_titlebar .rc_rate').text(i18n.getMessage("pidTuningRcRate"));
+                $('#pid-tuning .pid_titlebar .rate').text(i18n.getMessage("pidTuningRate"));
+                $('#pid-tuning .pid_titlebar .rc_expo').text(i18n.getMessage("pidTuningRcExpo"));
+
+                break;
+        }
+
+        if (!sameRatesType) {                    
+            $('.pid_tuning input[class="rate_input"]').val(rate_default);
+            $('.pid_tuning input[class="rc_rate_input"]').val(rc_rate_default);
+            $('.pid_tuning input[class="expo_input"]').val(expo_default);
+        }
+        
+        $('#pid-tuning input[class="rc_rate_input"]').attr({"max":rc_rate_max, "min":rc_rate_min, "step":rc_rate_step}).change();
+        $('#pid-tuning input[class="rate_input"]').attr({"max":rate_max, "min":rate_min, "step":rate_step}).change();
+        $('#pid-tuning input[class="expo_input"]').attr({"max":expo_max, "min":expo_min, "step":expo_step}).change();
+    
+        if (sameRatesType) {
+            self.setDirty(false);
+        }
+    }
+}
+
+TABS.pid_tuning.changeRatesTypeLogo = function() {
```

Comment:
While this looks great, I am not sure how much value it is, especially for users with small screens. So we might drop the rates logos at some point in the future.

Reply:
I think maybe they are too big. I like the logos but maybe a little smaller will fit in big and small screens.

Reply:
The logos occupy an unused space in the rates type selection tab, and scale accordingly, so no space wasted. They are a late addition to the code. If you see the screenshots you will clearly see that they are in empty and wasted space
![image](https://user-images.githubusercontent.com/35608074/76159916-a2b9ef00-6125-11ea-8856-56ed85e79058.png)
![image](https://user-images.githubusercontent.com/35608074/76159920-b5342880-6125-11ea-9dde-ca426c9927af.png)
If then there are problems they can be removed, but i think it's better this way
(And i personally tested the app all the time in a 1366x768 screen with the smallest possible windows size without problems)

Reply:
What I say is that it does not need to occupy all the space. Maybe smaller looks cleaner. But it always depends on the taste of the developer 😉

Reply:
Will do a little smaller then